modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -50,6 +51,14 @@ display-scope-facet=Display Site Facet
 display-selected-result-in-context=Display Selected Result in Context
 display-selected-result-in-context-help=Display the full context of a selected result when available.
 display-user-facet=Display User Facet
+document-creation-date=Document Creation Date
+document-creation-date-help=Display the document creation date.
+document-creator-user-name=Document Creator User Name Entry
```

Comment:
Is _Entry_ necessary here?

Reply:
Nope, I'll change it to `Document Creator User Name`.